### PR TITLE
Update no games message to give more context

### PR DESCRIPTION
**Pull request by rubenwardy**
I've seen a lot of user feedback where they don't understand what the message means. Hopefully adding some context will help

image

## To do

This PR is Ready for Review

## How to test

Read

### CODE REVIEW

**Comment by sfan5** [comment]
Somehow the text looks very alone in this big tab. How about making it bigger?

**Comment by grorp** [comment]
Vertical alignment is bad on my phone:

screenshot

I suggest to make the text area use all available vertical space between tabheader and "Install a game" button, then center the text vertically. Could use hypertext "valign" for this.

**Comment by rubenwardy** [comment]
image

image

**Comment by rubenwardy** [comment]
With size 18

image

image

**Review by sfan5 — APPROVED** [review]
lgtm (based on the screenshots)

File: builtin/mainmenu/tab_local.lua [hunk 1]
```diff
@@ -156,10 +156,18 @@ local function get_formspec(tabview, name, tabdata)
...
+				fgettext_ne("Minetest doesn't come with a game by default.") .. " " ..
+				fgettext_ne("You need to install a game before you can create a world.")
+
+		local button_y = H * 2/3 - 0.6
 		return table.concat({
-			"style[label_button;border=false]",
-			"button[2.75,1.5;10,1;label_button;", fgettext("You have no games installed."), "]",
-			"button[5.25,3.5;5,1.2;game_open_cdb;", fgettext("Install a game"), "]"})
+			"hypertext[0.375,0.375;", W - 2*0.375, ",", button_y, ";ht;", core.formspec_escape(hypertext), "]",
+			"button[5.25,", button_y, ";5,1.2;game_open_cdb;", fgettext("Install a game"), "]"})
```
Comment (grorp) [inline builtin/mainmenu/tab_local.lua:170]: I assume the intersection between button and hypertext is intentional for better vertical alignment?

**Review by grorp — APPROVED** [review]

File: builtin/mainmenu/tab_local.lua (hunk 1 above)
Comment (rubenwardy) [reply builtin/mainmenu/tab_local.lua:170]: no that's a mistake, forgot to subtract the 0.375

**Review by grorp — APPROVED** [review]

**Pull request merged**